<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManage</class>
 <widget class="QWidget" name="DeviceManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceManage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnAddDevice">
       <property name="text">
        <string>添加设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveDevice">
       <property name="text">
        <string>删除设备</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
